UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteDataSO.cs

```diff
@@ -12,6 +12,8 @@ public class SpriteDataSO : ScriptableObject
 	public bool IsPalette = false;
 	public int setID = -1;
 
+	public string DisplayName;
```

Comment:
why not just rename file?

Reply:
1. Underwear, Undershirts, etc already have some sort of display name that let's them display correctly in game while hair, facial hair, lizard tails, etc don't. This makes it so everything is equal.
2. Because that's a terrible fix.

Reply:
Uses less obscure names and Makes it easy to find in project I don't get what the problem is